src/style/variables.sass

```
@@ -62,8 +63,10 @@ $extra-large-screen: 1200px
 
 // Fonts
 @import url('https://fonts.googleapis.com/css2?family=Heebo:wght@400;500&display=swap')
+@import url('https://fonts.googleapis.com/css2?family=JetBrains+Mono:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800&display=swap')
```

The import requests should be combined (extra HTTP request). Also, there isn't a need in all the font-weights of `JetBrains Mono`. Only need font-weight 400.

---

Here's a suggestion:
```suggestion
@import url('https://fonts.googleapis.com/css2?family=Heebo:wght@400;500&family=JetBrains+Mono&display=swap')
```